dali/kernels/tensor_shape.h

```diff
@@ -436,6 +436,15 @@ flatten_shapes(const std::vector<T> &shapes) {
   return result;
 }
 
+static int get_dim_from_uniform(std::initializer_list<std::vector<int64_t>> shapes) {
+  return get_dim_from_uniform(std::vector<std::vector<int64_t>>(shapes));
+}
+
+static std::vector<int64_t> flatten_shapes(std::initializer_list<std::vector<int64_t>> shapes) {
```

Comment:
Wasn't that supposed to be SmallVector?

Reply:
SmallVector is not compatible with vector and causes problems with two arguments provided, trying to use the constructor for `begin` and `end` iterators, which are not constrained to be InputIterators (but are a template accepting any type). Will leave until this is fixed.